core/templates/components/summary-tile/collection-summary-tile.component.ts

```diff
@@ -30,24 +30,24 @@ import { UserService } from 'services/user.service';
   templateUrl: './collection-summary-tile.component.html',
 })
 export class CollectionSummaryTileComponent implements OnInit {
-  @Input() getCollectionId: string;
-  @Input() getCollectionTitle: string;
-  @Input() getLastUpdatedMsec: number;
-  @Input() getObjective: string;
-  @Input() getNodeCount: string;
-  @Input() getCategory: string;
-  @Input() getThumbnailIconUrl: string;
-  @Input() getThumbnailBgColor: string;
-  @Input() isPlaylistTile: boolean;
-  @Input() isLinkedToEditorPage: boolean;
-  @Input() showLearnerDashboardIconsIfPossible: string;
-  @Input() isContainerNarrow: boolean;
-  @Input() isOwnedByCurrentUser: boolean;
+  @Input() getCollectionId!: string;
+  @Input() getCollectionTitle!: string;
+  @Input() getLastUpdatedMsec!: number;
+  @Input() getObjective!: string;
+  @Input() getNodeCount!: string;
+  @Input() getCategory!: string;
+  @Input() getThumbnailIconUrl!: string;
+  @Input() getThumbnailBgColor!: string;
+  @Input() isPlaylistTile!: boolean;
+  @Input() isLinkedToEditorPage!: boolean;
+  @Input() showLearnerDashboardIconsIfPossible!: string;
+  @Input() isContainerNarrow!: boolean;
+  @Input() isOwnedByCurrentUser!: boolean;
 
-  userIsLoggedIn: boolean;
-  collectionIsCurrentlyHoveredOver: boolean;
-  defaultEmptyTitle: string;
-  activityTypeCollection: string;
+  userIsLoggedIn!: boolean;
```

Comment:
why can this never be null or undefined?

Reply:
I think that there is no point of making a boolean, null or undefined because that does also mean it to be false. Boolean values should be either true or false only, there no usage of `!` operator.